config/betterquesting/DefaultQuests/Quests/CoinsCoinsCoins-AAAAAAAAAAAAAAAAAAAAEA==/DamascusSteelIng-AAAAAAAAAAAAAAAAAAAEkw==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Damascus Steel is very durable and useful for GT tools. Buy some."
+      "desc:8": "Damascus steel is very durable and useful for GT tools. Buy some."
```

Comment:
Should probably be "Buy some!" to conform with the others rather than "Buy some.".

Reply:
Changed